### ISSUE TITLE
No module named 'nnunet.network_architecture.multi_modal'

### ISSUE THREAD

**Issue by shanwq** [Zirui0623/A2FSeg#2]
Hi, your work is interesting! When I followed your read.me file, after the data preparation, I am trying to do the train procedure, there is an error occurred: No module named 'nnunet.network_architecture.multi_modal'.  
The code:'from nnunet.network_architecture.multi_modal.utils import subset_idx' has appeared many times in your source code.
But there is not a multi_modal path under the path: nnunet/network_architecture/ . 
Why is that? Have you forgotten this file or have I missed something?

**Comment by Zirui0623** [comment]
Thanks for your attention. At present, the code has not been sorted out, and the multi_modal folder has just been uploaded. I've been busy recently, and I'll continue to improve it later. If you have any more questions, you can email me again.




------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2023年11月2日(星期四) 下午3:26
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; 
主题: [Zirui0623/A2FSeg] No module named &#39;nnunet.network_architecture.multi_modal&#39; (Issue #2)





 
Hi, your work is interesting! When I followed your read.me file, after the data preparation, I am trying to do the train procedure, there is an error occurred: No module named 'nnunet.network_architecture.multi_modal'.
 The code:'from nnunet.network_architecture.multi_modal.utils import subset_idx' has appeared many times in your source code.
 But there is not a multi_modal path under the path: nnunet/network_architecture/ .
 Why is that? Have you forgotten this file or have I missed something?

**Comment by princerice** [comment]
May I ask  if your data set is brats2020? Does the data processing part need to change some operations

**Comment by princerice** [comment]
> Hi, your work is interesting! When I followed your read.me file, after the data preparation, I am trying to do the train procedure, there is an error occurred: No module named 'nnunet.network_architecture.multi_modal'. The code:'from nnunet.network_architecture.multi_modal.utils import subset_idx' has appeared many times in your source code. But there is not a multi_modal path under the path: nnunet/network_architecture/ . Why is that? Have you forgotten this file or have I missed something?

May I ask if your data set is brats2020? Does the data processing part need to change some operations

**Comment by Zirui0623** [comment]
Yes, the dataset I use is brats2020, and the default processing method of nnunet is used in the data preprocessing part.




------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2023年11月13日(星期一) 上午9:49
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [Zirui0623/A2FSeg] No module named &#39;nnunet.network_architecture.multi_modal&#39; (Issue #2)





  
Hi, your work is interesting! When I followed your read.me file, after the data preparation, I am trying to do the train procedure, there is an error occurred: No module named 'nnunet.network_architecture.multi_modal'. The code:'from nnunet.network_architecture.multi_modal.utils import subset_idx' has appeared many times in your source code. But there is not a multi_modal path under the path: nnunet/network_architecture/ . Why is that? Have you forgotten this file or have I missed something?
  
May I ask if your data set is brats2020? Does the data processing part need to change some operations

**Comment by princerice** [comment]
> 是的，我使用的数据集是brats2020，数据预处理部分使用了nnunet的默认处理方式。
> [...](#)
> ------------------&nbsp;原始邮件&nbsp;------------------ 发件人: ***@***.***&gt;; 发送时间: 2023年11月13日(星期一) 上午9:49 收件人: ***@***.***&gt;; 抄送: ***@***.***&gt;; ***@***.***&gt;; 主题: Re: [Zirui0623/A2FSeg] No module named &#39;nnunet.network_architecture.multi_modal&#39; (Issue #2) Hi, your work is interesting! When I followed your read.me file, after the data preparation, I am trying to do the train procedure, there is an error occurred: No module named 'nnunet.network_architecture.multi_modal'. The code:'from nnunet.network_architecture.multi_modal.utils import subset_idx' has appeared many times in your source code. But there is not a multi_modal path under the path: nnunet/network_architecture/ . Why is that? Have you forgotten this file or have I missed something? May I ask if your data set is brats2020? Does the data processing part need to change some operations — Reply to this email directly, view it on GitHub, or unsubscribe. You are receiving this because you commented.Message ID: ***@***.***&gt;

Ok, thanks for the author's reply. Run down the code and try it with the brats dataset. I'll leave you a message if I have any other questions
